google-compute-engine/src/test/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceLiveTest.java

```diff
@@ -67,6 +74,34 @@ public void testListHardwareProfiles() throws Exception {
       }
    }
 
+   public void testCreateNodeWithSsd() throws Exception {
+      String group = this.group + "ssd";
+      try {
+         TemplateOptions options = client.templateOptions();
+
+         options.as(GoogleComputeEngineTemplateOptions.class).bootDiskType("pd-ssd");
```

Comment:
Hmm - actually, this fits more like machine type here in GCE than it does, say, instance type on EC2 - there's an API for getting the possible disk types, after all. So I'm really not sure whether we want to be hardcoding the names of disk types when they could change on the fly...